jwst/transforms/models.py

```diff
@@ -813,6 +813,122 @@ def __call__(self, v2, v3):
         return self.prepare_outputs(format_info, *outputs)
 
 
+class IdealToV2V3(Model):
+    """
+    Performs the transform from Ideal to telescope V2,V3 coordinate system.
+    The two systems have the same origin - V2_REF, V3_REF.
+
+    Note: This model has no schema implemented - add schema if needed.
+    """
+    
+    inputs = ('xidl', 'yidl')
+
+    outputs = ('v2', 'v3')
+
+    v3idlyangle = Parameter() # in deg
+    v2ref = Parameter() # in arcsec
+    v3ref = Parameter() # in arcsec
+    vparity = Parameter()
+
+    
+    def __init__(self, v3idlyangle, v2ref, v3ref, vparity, name='idl2V', **kwargs):
+        super(IdealToV2V3, self).__init__(v3idlyangle=v3idlyangle, v2ref=v2ref,
+                                          v3ref=v3ref, vparity=vparity, name=name,
+                                          **kwargs)
+
+    @staticmethod
+    def evaluate(xidl, yidl, v3idlyangle, v2ref, v3ref, vparity):
+        """
+        Parameters
+        ----------
+        xidl, yidl : ndarray-like
+            Coordinates in Ideal System [in deg]
+        v3idlyangle : float
+            Angle between Ideal Y-axis and V3 [ in deg]
+        v2ref, v3ref : ndarray-like
+            Coordinates in V2, V3 [in deg]
+        vparity : int
+            Parity.
+
+        Returns
+        -------
+        v2, v3 : ndarray-like
+            Coordinates in the (V2, V3) telescope system [in deg].
+
+        """
+        v2ref = np.deg2rad(v2ref / 3600)
+        v3ref = np.deg2rad(v3ref / 3600)
+        v3idlyangle = np.deg2rad(v3idlyangle)
+        xidl = np.deg2rad(xidl)
+        yidl = np.deg2rad(yidl)
```

Comment:
For the case that we need this transform (converting the XIdl, YIdl values from PPS) the inputs will be in units of arcsec. Here you're assuming they're in degrees. Will the inputs be converted from arcsec to degrees before being sent to this function?